packages/terra-date-time-picker/src/_TimeClarification.jsx

```diff
@@ -55,6 +55,11 @@ const propTypes = {
    * Whether the clarification is disabled.
    */
   disabled: PropTypes.bool,
+  /**
+   * Timezone value to indicate in which timezone the date-time component is rendered.
+   * The value provided should be a valid [timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) string, else will default to browser/local timezone.
+   */
+  timeZone: PropTypes.string,
```

Comment:
Should this also be initalTimeZone?

Reply:
Apart from the prop exposed to consumers, I have retained everywhere as "timeZone". If the prop in `_TimeClarification.jsx` confusing, I will go ahead and change it it `initialTimeZone`.

Reply:
IMO we should keep the naming consistent if the functionality is constant throughout even if it's passes to internal components. If others think it's fine to leave it as it, I'm okay with keeping it.

Reply:
I also vote to make them consistent to minimize any confusion.

Reply:
Changed here 3cadcdb1ff2efc1ba7f4eef489e4bad1f080be6d